ui/signup.html

```diff
@@ -0,0 +1,175 @@
+                                                </a>
+                                            </span>
+                                            <span class="recaptcha_only_if_image">
+                                                <a class="btn" href="javascript:Recaptcha.switch_type('audio')" title="__MSG__GET_AN_AUDIO_CHALLENGE__">
+                                                    <i class="fa fa-volume-up"></i>
+                                                </a>
+                                            </span>
+                                            <span class="recaptcha_only_if_audio">
+                                                <a class="btn" href="javascript:Recaptcha.switch_type('image')" title="__MSG__GET_A_VISUAL_CHALLENGE__">
+                                                    <i class="fa fa-font"></i>
+                                                </a>
+                                            </span>
+                                        </div>
+                                        <div class="signup-createaccount-captcha-input-container">
+                                            <label for="recaptcha_response_field" class="control-label sr-only">__MSG__ENTER_THE_TEXT__</label>
+                                            <input type="text" id="recaptcha_response_field" class="form-control" autocapitalize="off" name="recaptcha_response_field" placeholder="__MSG__ENTER_THE_TEXT__"/>
+                                        </div>
+                                        <label id="signup-createaccount-captcha-error" class="hide help-block" for="recaptcha_response_field">__MSG__CAPTCHA_ERROR__</label>
+                                    </div>
+                                </div>
+                            {/if}
+                            {if termsAndConditionsEnabled}
+                                <div id="signup-createaccount-termsandconditions-container" class="form-group">
+                                    <div class="col-xs-12 col-md-7 pull-right">
+                                        {var termsAndConditionsLink = '<a href="#" role="button" class="oae-trigger-termsandconditions">__MSG__TERMS_AND_CONDITIONS__</a>'}
+                                        <label for="signup-createaccount-accept-terms-and-conditions" class="checkbox control-label">
+                                            <input type="checkbox" id="signup-createaccount-accept-terms-and-conditions" class="required" name="termsandconditions"/>__MSG__I_AGREE_TO_THE_TERMS_AND_CONDITIONS__
+                                        </label>
+                                        <div class="help help-block"></div>
+                                    </div>
+                                </div>
+                            {/if}
+                            <div class="form-group">
+                                <div class="col-xs-12 col-md-7 pull-right">
```

Comment:
It's needed because the signup button has to be aligned with the fields and not the labels of the form